relay-server/src/services/processor/event.rs

```diff
@@ -420,6 +420,26 @@ pub fn has_unprintable_fields(event: &Annotated<Event>) -> bool {
     }
 }
 
+//TODO: comment
+//TODO: add test: all incr/no incr cases, envelope unchanged
```

Comment:
Will add test coverage after we're aligned on whether to emit the metric here, placement of the helper fx, etc

Reply:
I am fine with not having tests for metrics. A comment would be nice though